app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +23,8 @@ public function boot()
      */
     public function register()
     {
-        //
+        if ($this->app->environment('development')) {
```

Comment:
Hmm, that is right according to the docs:
https://laravel.com/docs/5.2/configuration#determining-the-current-environment